Assets/LeapMotion/Core/Scripts/XR/Editor/XRHeightOffsetEditor.cs

```diff
@@ -43,8 +48,12 @@ public class XRHeightOffsetEditor : CustomEditorBase<XRHeightOffset> {
     //}
 
     private bool isRoomScaleTrackingDetected() {
```

Comment:
Any reason this still needs to exist?  Should this not be replaced with XRSupportUtils.IsRoomScale ?